python/cugraph/cugraph/centrality/betweenness_centrality.py

```diff
@@ -106,10 +106,8 @@ def betweenness_centrality(
 
     Examples
     --------
-    >>> gdf = cudf.read_csv(datasets_path / 'karate.csv', delimiter=' ',
```

Comment:
I'm thinking we can remove [this line](https://github.com/rapidsai/cugraph/blob/2a57740c8eac484ddd1a200f0e93f0dbb4062679/python/cugraph/cugraph/tests/test_doctests.py#L153) now too, since the docstrings won't be using the `dataset_path` variable.  Removing that will also make sure we didn't forget anything, since the old docstring pattern will fail.

Reply:
There are some docstrings that still rely on this path and I believe shouldn't be replaced by the datasets API. For example, there are some MG examples in the `cugraph/dask` directory, as well as functions like `from_cudf_edgelist` in `cugraph/structures` that demonstrate more granular behavior that shouldn't be overwritten by the datasets API

Reply:
Removing the line in `test_doctests.py` causes these tests to fail unless the `datasets_path` is given to them elsewhere, or included in the respective docstring example.

Reply:
> functions like from_cudf_edgelist in cugraph/structures that demonstrate more granular behavior that shouldn't be overwritten by the datasets API

That's a good reason to keep it.